### DISCUSSION TITLE
I also wanted to learn about workflow

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#276] [Q&A]
Hey,
I wanted to learn about workflow how to use, types of workflow and many more so i can been able to handle with workflow problem  can anyone tell be few better resource from where i can learn 
Thanks

**Comment by ming-tsai** [comment]
Do you mean Github Actions?

**Reply by UG-SEP** [reply]
yes @ming-tsai

**Reply by ming-tsai** [reply]
I always read the [documentation](https://docs.github.com/es/actions), that it.
